packages/relay/src/lib/poller.ts

```diff
@@ -87,7 +88,10 @@ export class Poller {
 
           poll.lastPolled = this.latestBlock;
         } else if (event === this.NEW_HEADS_EVENT && this.newHeadsEnabled) {
-          data = await this.eth.getBlockByNumber('latest', filters?.includeTransactions ?? false);
+          data = await this.eth.getBlockByNumber('latest', filters?.includeTransactions ?? false, {
+            requestIdPrefix: '',
+            requestIp: '',
+          } as IRequestDetails);
```

Comment:
In those cases, `undefined` would be better instead of an object with all empty strings because it strongly suggests that this information is not available. 

```suggestion
          data = await this.eth.getBlockByNumber('latest', filters?.includeTransactions ?? false);
```

Reply:
The idea was to have the request id everywhere and not have the undefined as an option, so maybe its better to generate request id

Reply:
I have changed it so we generate a requestId